docs/manifest.json

```diff
@@ -29,6 +29,12 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/block-edit-save.md",
 		"parent": "block-api"
 	},
+	{
+		"title": "Editable API",
+		"slug": "editable-api",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/blocks\/editable\/readme.md",
```

Comment:
Should we move this document to `docs` folder and reference it from `editable` readme file? It might be good for visibility when someone explores docs inside git repository. Just an idea, not a blocker for sure.

Reply:
Hm, I'm not sure, I think it's good to have docs inside the component so it's updated on changes. I don't mind either way though, it can be referenced from either side. I'm open to change in the future.

Reply:
Sounds good 👍